F/FLINT/build_tarballs.jl

```diff
@@ -52,6 +52,18 @@ for f in ${WORKSPACE}/srcdir/patches/*.patch; do
   atomic_patch -p1 ${f}
 done
 
+if [[ "$nbits" == "64" ]] && [[ "$target" != aarch64-* ]]; then
```

Comment:
Note that this won't work in Julia v1.7, where we do ILP64 for aarch64 platforms :disappointed:

Reply:
Is there an example on Yggdrasil where this is properly done?

Reply:
Ehm.... :see_no_evil:

Reply:
Elsewhere I suggested using OpenBLAS32, which has the twfold advantage of non requiring renaming all symbols and not having this inconvenient split 1.6/1.7

Reply:
@tthsqe12 Do you care that we will have 32-bit BlasInt? If not `OpenBLAS32` might be an option. See https://github.com/JuliaPackaging/Yggdrasil/pull/697

Reply:
Seems to work here and it worked my box. 🤷

Reply:
Patches aren't needed anymore if you're targeting OpenBLAS32